artifactor/plugins/reporter.py

```diff
@@ -267,153 +264,203 @@ def build_dict(self, path, container, contents):
         if not end:
-            container['_sub'][head] = contents
-            container['_stats'][contents['outcomes']['overall']] += 1
-            container['_duration'] += contents['duration']
+            container["_sub"][head] = contents
+            container["_stats"][contents["outcomes"]["overall"]] += 1
+            container["_duration"] += contents["duration"]
         # If we are in a module.
         else:
-            if head not in container['_sub']:
-                container['_sub'][head] = deepcopy(_tests_tpl)
+            if head not in container["_sub"]:
+                container["_sub"][head] = deepcopy(_tests_tpl)
             # Call again to recurse down the tree.
-            self.build_dict(end, container['_sub'][head], contents)
-            container['_stats'][contents['outcomes']['overall']] += 1
-            container['_duration'] += contents['duration']
+            self.build_dict(end, container["_sub"][head], contents)
+            container["_stats"][contents["outcomes"]["overall"]] += 1
+            container["_duration"] += contents["duration"]
 
     def build_li(self, lev):
         """
         Build up the actual HTML tree from the dict from build_dict
         """
-        bimdict = {'passed': 'success',
-                   'failed': 'warning',
-                   'error': 'danger',
-                   'skipped': 'primary',
-                   'xpassed': 'danger',
-                   'xfailed': 'success'}
-        list_string = '<ul>\n'
-        for k, v in lev['_sub'].items():
+        bimdict = {
+            "passed": "success",
+            "failed": "warning",
+            "error": "danger",
+            "skipped": "primary",
+            "xpassed": "danger",
+            "xfailed": "success",
+        }
+        list_string = "<ul>\n"
+        for k, v in lev["_sub"].items():
 
             # If 'name' is an attribute then we are looking at a test (leaf).
-            if 'name' in v:
-                pretty_time = str(datetime.timedelta(seconds=math.ceil(v['duration'])))
+            if "name" in v:
+                pretty_time = str(datetime.timedelta(seconds=math.ceil(v["duration"])))
                 teststring = '<span name="mod_lev" class="label label-primary">T</span>'
                 label = '<span class="label label-{}">{}</span>'.format(
-                    bimdict[v['outcomes']['overall']], v['outcomes']['overall'].upper())
-                proc_name = process_pytest_path(v['name'])[-1]
+                    bimdict[v["outcomes"]["overall"]], v["outcomes"]["overall"].upper()
+                )
+                proc_name = process_pytest_path(v["name"])[-1]
                 link = (
-                    '<a href="#{}">{} {} {} <span style="color:#888888"><em>[{}]</em></span></a>'
-                    .format(v['name'], proc_name, teststring, label, pretty_time))
+                    '<a href="#{}">{} {} {} ' '<span style="color:#888888"><em>[{}]</em></span></a>'
```

Comment:
This doesn't look quite right, why is it injecting single quotes here

Reply:
double quotes in html text

Reply:
please explain

Reply:
actually i missunderstood what was happening at first glance
this is potentially a bug, but could aslo be a rebase artifact